src/evaluators/utility/QCAD_EvaluatorTools.hpp

```diff
@@ -75,6 +75,20 @@ namespace QCAD
     std::string getEvalType() const;
   };
 
+  // HessianVec
+  template<typename Traits>
+  class EvaluatorTools<PHAL::AlbanyTraits::HessianVec, Traits>
```

Comment:
Here's another class that I'm not sure we should maintain.

Reply:
Same answer as above.